src/components/atoms/TextInput.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from "react";
```

Comment:
Very nice addition, this component :) 
Consider placing this file in a folder in the components folder that is called Input.

Reply:
Good idea ;)